apps/challenges/challenge_config_utils.py

```diff
@@ -370,6 +388,23 @@ def validate_challenge_config_util(
             error_messages.append(message)
             return error_messages, yaml_file_data, files
 
+    # Get existing config IDs for leaderboards and dataset splits
+    if existing_challenge:
+        existing_challenge_phases = ChallengePhase.objects.filter(challenge=existing_challenge.id)
+        existing_challenge_phase_splits = ChallengePhaseSplit.objects.filter(challenge_phase__in=existing_challenge_phases)
+        existing_leaderboards = Leaderboard.objects.filter(id__in=existing_challenge_phase_splits.values('leaderboard'))
+        existing_dataset_splits = DatasetSplit.objects.filter(id__in=existing_challenge_phase_splits.values('dataset_split'))
+
+        existing_leaderboard_config_ids = [int(x.config_id) for x in existing_leaderboards]
+        existing_dataset_split_config_ids = [int(x.config_id) for x in existing_dataset_splits]
+        existing_challenge_phase_config_ids = [int(x.config_id) for x in existing_challenge_phases]
+        existing_challenge_phase_split_id_triples = [(split.leaderboard.config_id, split.challenge_phase.config_id, split.dataset_split.config_id) for split in existing_challenge_phase_splits]
```

Comment:
rename variable and make code readable

Reply:
I have updated it but if you have any suggestions please let me know.